src/PaylineProvider.tsx

```diff
@@ -1,27 +1,20 @@
-import React, { useLayoutEffect, useState } from 'react';
+import React, { useInsertionEffect, useState } from 'react';
 
 type PropsType = {
   production?: boolean;
 };
 
-export const PaylineHead: React.ComponentType<PropsType> = ({ production = false }) => {
```

Comment:
Why are you removing PaylineHead component? Even if not used, does it conflict with SSR?